### ISSUE TITLE
[Accessibility] Toggle component should take ariaLabel for the checkbox button

### ISSUE THREAD

**Issue by yaananth** [OfficeDev/office-ui-fabric-react#1136]
### Bug Report

- __Package version(s)__: latest

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes)

Requested priority: (Normal) 

#### Describe the issue:

https://dev.office.com/fabric#/components/toggle
If someone uses a screen reader and skips to buttons, for Toggle component, it would just say it's just a checkbox, authors should be able to send some aria-labels to it so that screen reader can read out what that checkbox is for. (It is important for the button to have the context, not everyone goes through the page element by element, we can't expect that labels that are sibling to buttons can take care of it)

**Comment by yaananth** [comment]
dup of #1092
